extensions/amp-form/0.1/test/test-form-validators.js

```diff
@@ -123,22 +124,78 @@ describes.realWin('form-validators', {amp: true}, env => {
     });
   });
 
+  describe('FormValidator', () => {
+    let form, validator;
+
+    beforeEach(() => {
+      const doc = env.win.document;
+      form = getForm(doc);
+      validator = new FormValidator(form);
```

Comment:
True, this would fail if we ran Closure compiler over test code. But I think this would be the least of Closure's complaints. 😄